db/seeds.rb

```diff
@@ -175,6 +175,13 @@ def setup_application_instances(application, application_instances)
   end
 end
 
+BundleInstance.all.each do |bundle_instance|
```

Comment:
Always use `.find_each` instead. Doing `.all.each` will load every single record into memory. Which isn't bad when you are local, but as soon as you have 100,000 records on prod...that really hurts and will likely cause this to blow up. `.find_each` will, by default, load 1,000 records at a time.